packages/sdk-core/src/Operation.ts

```diff
@@ -54,12 +57,27 @@ export default class Operation {
      * @returns {Promise<TransactionRequest>}
      */
     getPopulatedTransactionRequest = async (
-        signer: ethers.Signer
+        signer: ethers.Signer,
+        gasLimitMultiplier = 1.2
     ): Promise<TransactionRequest> => {
         const txnToPopulate = this.forwarderPopulatedPromise
             ? await this.forwarderPopulatedPromise
             : await this.populateTransactionPromise;
-        return await signer.populateTransaction(txnToPopulate);
+        const signerPopulatedTransaction = await signer.populateTransaction(
+            txnToPopulate
+        );
+
+        return {
+            ...signerPopulatedTransaction,
+            gasLimit:
```

Comment:
So if I explicitly passed in a gas limit with `Overrides` then this thing will multiply it?

Reply:
yeah, prolly shouldn't be doing this given the name `Overrides`.